### DISCUSSION TITLE
How can I ban the use of validation_step in the traning stage

### DISCUSSION THREAD

**Discussion by chenxn2020** [Lightning-AI/pytorch-lightning#10563] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hello:
I want to use trainer.fit() to train model only  without doing validation_step. How can I reach my goal？ 
The approach I currently think of is to set the  check_val_every_n_epoch very large

**Comment by rohitgr7** [comment]
you can just set `Trainer(..., limit_val_batches=0)`

**Reply by chenxn2020** [reply]
Thanks for your reply, I get it
